aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -920,7 +922,7 @@ class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknownHtmlT
                     }
                 }
             }
-            videoThumbnailGetter?.loadVideoThumbnail(it.getSource(), callbacks, this@AztecText.maxImagesWidth, this@AztecText.minImagesWidth)
+            videoThumbnailGetter?.loadVideoThumbnail(it.getSource(), callbacks, maxDimension, maxDimension)
```

Comment:
same here, we can get rid of `val maxDimension` and use `maxImagesWidth` instead, for both max and min parameters in this call